### PR TITLE
testHexCCConverter test correction when magic quotes is on

### PR DESCRIPTION
**Pull request by Awnage**
Unit test fix for when magic quotes is on to make sure impact is correct.
Two Spelling corrections in the filters

### CODE REVIEW

File: tests/IDS/Tests/MonitorTest.php [hunk 1]
```diff
@@ -1290,7 +1290,12 @@ public function testHexCCConverter()
             $this->init
         );
         $result = $test->run($exploits);
-        $this->assertEquals(109, $result->getImpact());
+        if (function_exists('get_magic_quotes_gpc') and @get_magic_quotes_gpc()) {
+            $this->assertEquals(116, $result->getImpact());
+        }
+        else{
```
Comment (jayzeng) [inline tests/IDS/Tests/MonitorTest.php]: should add a space after else to be consistent

**Comment by lstrojny** [comment]
Very good improvement. Two things:
- Let’s add a parametrized build with magic quotes enabled
- Let’s not use the shutdown operator but instead check for the appropriate PHP version

**Comment by Awnage** [comment]
Good point. I just compiled PHP5.5b1 and had no failing tests. 5.3.15 was generating the failed tests. Given that (according to Automatic) 5.3 usage is still at 34%, might be good to check versions.

**Comment by lstrojny** [comment]
What about actually doing strip slashes on the request depending on the version and configuration settings?

**Comment by Awnage** [comment]
I have only looked at the unit tests so far. This is my first real jump into digging into PHPIDS and trying to help out. Want me to give that a shot?

**Comment by lstrojny** [comment]
Why was this closed?

**Comment by Awnage** [comment]
I am fairly new to git and how pushes/pulls work. I closed it for 2 reasons: 
  the last commit I made breaks the unit tests
  the discussion veered away from the simple commit I was trying to make in the first place.

**Pull request closed without merging**